docs/reverse-replication/ReverseReplicationUserGuide.md

```diff
@@ -45,7 +45,7 @@ These steps are achieved by two Dataflow jobs, along with an interim buffer whic
 
 A few prerequisites must be considered before starting with reverse replication.
 
-1. Make sure that there is network connectivity between source database and your GCP project on which the Dataflow jobs will run.Ensure the Dataflow worker IPs can access the MySQL IPs.
+1. Make sure that there is network connectivity between source database and your GCP project on which the Dataflow jobs will run.Ensure the Dataflow worker IPs can access the MySQL IPs, meaning Dataflow IPs are allowlisted on the MySQL instance. The MySQL credentials are correctly specidied in the [source shards file](./RunnigReverseReplication.md#sample-sourceshards-file) and MySQL server  is up. 
```

Comment:
How about this:
1. Ensure network connectivity between the source database and your GCP project, where your Dataflow jobs will run.
    - Allowlist Dataflow worker IPs on the MySQL instance so that they can access the MySQL IPs.
    - Check that the MySQL credentials are correctly specified in the [source shards file](./RunningReverseReplication.md#sample-sourceshards-file).
    - Check that the MySQL server is up.

Reply:
Done